### ISSUE TITLE
[BUG] submodule sync failed to find nvcomp

### ISSUE THREAD

**Issue by pxLi** [NVIDIA/spark-rapids-jni#2465]
**Describe the bug**
spark-rapids-jni_submodule-sync-dev, run: 3776

failed below with cudf change https://github.com/rapidsai/cudf/pull/16946
```
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   By not providing "Findnvcomp.cmake" in CMAKE_MODULE_PATH this project has
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   asked CMake to find a package configuration file provided by "nvcomp", but
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   CMake did not find one.
[2024-10-02T20:24:57.355Z] [INFO]      [exec] 
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   Could not find a package configuration file provided by "nvcomp" with any
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   of the following names:
[2024-10-02T20:24:57.355Z] [INFO]      [exec] 
[2024-10-02T20:24:57.355Z] [INFO]      [exec]     nvcompConfig.cmake
[2024-10-02T20:24:57.355Z] [INFO]      [exec]     nvcomp-config.cmake
[2024-10-02T20:24:57.355Z] [INFO]      [exec] 
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   Add the installation prefix of "nvcomp" to CMAKE_PREFIX_PATH or set
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   "nvcomp_DIR" to a directory containing one of the above files.  If "nvcomp"
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   provides a separate development package or SDK, be sure it has been
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   installed.
[2024-10-02T20:24:57.355Z] [INFO]      [exec] Call Stack (most recent call first):
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   /home/jenkins/agent/workspace/jenkins-spark-rapids-jni_submodule-sync-dev-3776/target/libcudf-install/lib64/cmake/cudf/cudf-dependencies.cmake:53 (find_dependency)
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   /home/jenkins/agent/workspace/jenkins-spark-rapids-jni_submodule-sync-dev-3776/target/libcudf-install/lib64/cmake/cudf/cudf-config.cmake:90 (include)
[2024-10-02T20:24:57.355Z] [INFO]      [exec]   /home/jenkins/agent/workspace/jenkins-spark-rapids-jni_submodule-sync-dev-3776/target/jni/cmake-build/_deps/rapids-cmake-src/rapids-cmake/find/package.cmake:125 (find_package)
[2024-10-02T20:24:57.356Z] [INFO]      [exec]   CMakeLists.txt:146 (rapids_find_package)
```

**Steps/Code to reproduce bug**
Please provide a list of steps or a code sample to reproduce the issue.
Avoid posting private or sensitive data.

**Expected behavior**
Pass the build

**Environment details (please complete the following information)**
 - Environment location: [Standalone, YARN, Kubernetes, Cloud(specify cloud provider)]
 - Spark configuration settings related to the issue

**Additional context**

**Comment by jlowe** [comment]
Filed https://github.com/rapidsai/cudf/issues/16986

**Comment by jlowe** [comment]
Fixed by rapidsai/cudf#16988
